src/pages/Encounters/EncounterList.tsx

```
@@ -161,7 +161,19 @@ export function EncounterList({
+  const handleFieldChange = () => {
+    {
+      updateQuery({
+        status,
+        encounter_class: encounterClass,
+        priority,
+        name: undefined,
+        encounter_id: undefined,
+        external_identifier: undefined,
+      });
+      clearSearch.value = true;
+    }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)